client/app/components/packages/pas-form/edit.hbs

```diff
@@ -1,97 +1,103 @@
 <SaveableForm
-  @model={{this.pasForm}}
-  @validators={{array this.validations.SaveablePasFormValidations this.validations.SubmittablePasFormValidations}}
-  as |saveableForm|
+  @model={{@package}}
+  @validators={{array hash this.validations.SubmittablePackageFormValidations}}
```

Comment:
why hash here?

Reply:
the component requires 2 validators and we don't have any saveability validations for packages